sockeye/inference.py

```diff
@@ -46,6 +47,9 @@ class InferenceModel(model.SockeyeModel):
     :param checkpoint: Checkpoint to load. If None, finds best parameters in model_folder.
     :param softmax_temperature: Optional parameter to control steepness of softmax distribution.
     :param max_output_length_num_stds: Number of standard deviations as safety margin for maximum output length.
+    :param decoder_return_logit_inputs: Decoder returns inputs to logit computation instead of softmax over target
+                                        vocabulary.  Used when logits/softmax are handled separately.
+    :param cache_output_layer_w_b: Cache weights and biases for logit computation as NumPy arrays.
```

Comment:
weights and biases below are mxnet ndarrays and not numpy arrays as mentioned in the comment, right?

Reply:
Fixed.